application/src/main/java/org/opentripplanner/standalone/server/DefaultServerRequestContext.java

```diff
@@ -253,6 +258,11 @@ public ViaCoordinateTransferFactory viaTransferResolver() {
     return viaTransferResolver;
   }
 
+  @Override
+  public TriasApiConfig triasApiConfig() {
```

Comment:
TriasApiConfig -> TriasApiParameters

Reply:
Would you like me to simply rename the class or create a new `TriasParameters` which is created by a mapper from the `TriasConfig`?

Reply:
5166c2776a6c8c4df3b7ca8f1b9e31ad342eb00b